### PR TITLE
Safe frame resize hack

### PR DESCRIPTION
**Pull request by pm-shashank-jain**
Resize of safe frame from outside by finding that particular iframe using divid we get using slotid

### CODE REVIEW

File: src_new/util.js [hunk 1]
```diff
@@ -324,11 +324,14 @@ exports.forEachGeneratedKey = function(adapterID, adUnits, adapterConfig, impres
...
 
-exports.resizeWindow = function(theDocument, width, height){
+exports.resizeWindow = function(theDocument, width, height,divId){
 	/* istanbul ignore else */
 	if(height && width){
 		try{
 			var fr = theDocument.defaultView.frameElement;
+			if(divId){
+			 	 fr = document.getElementById(divId).querySelector('iframe')
+			}
```
Comment (pm-harshad-mane) [inline src_new/util.js]: https://github.com/prebid/Prebid.js/commit/5b4e3828111dd3b9a6c5965fdcd67800c0bd838e
Refer above commit, we also need to resize the div

File: src_new/util.js [hunk 2]
```diff
@@ -324,11 +324,14 @@ exports.forEachGeneratedKey = function(adapterID, adUnits, adapterConfig, impres
...
 			fr.width = width;
 			fr.height = height;
 		}catch(e){} // eslint-disable-line no-empty
```
Comment (pm-harshad-mane) [inline src_new/util.js]: Log suitable errors

File: src_new/util.js (hunk 1 above)
Comment (pm-harshad-mane) [reply src_new/util.js]: @pm-shashank-jain  looks like you have missed this coment

File: src_new/util.js (hunk 2 above)
Comment (pm-harshad-mane) [reply src_new/util.js]: @pm-shashank-jain looks like you have missed this coment

File: src_new/util.js (hunk 1 above)
Comment (pm-shashank-jain) [reply src_new/util.js]: @pm-harshad-mane i did those changes

Comment (pm-harshad-mane) [reply src_new/util.js]: Cool, done.

**Pull request merged**